### ISSUE TITLE
Can you delete trailing whitespaces on cheat sheet.

### ISSUE THREAD

**Issue by eorroe** [MiroHibler/sublime-keymaps#21]
Trailing White spaces show up on cheat sheet.

**Comment by MiroHibler** [comment]
Yeah, I know it's annoying but it's there because of (centre) alignment and shows only if you installed some package that highlights them.

Definitely on my task list ;)

**Comment by eorroe** [comment]
Alright thanks.

**Comment by pykong** [comment]
At least the new Markdown Template render without trailing spaces:

![image](https://user-images.githubusercontent.com/20211140/75261913-0a01a600-57ec-11ea-8015-4ba588457014.png)
